keyboards/pinky/4/keymaps/via/rules.mk

```diff
@@ -0,0 +1,6 @@
+BOOTLOADER = atmel-dfu
```

Comment:
This will break reset capability for nearly everyone running this board on a Pro Micro if included. If you're using a controller that isn't using Caterina for its bootloader, this modification should be in a user keymap, separate from the VIA keymap.

```suggestion
```